pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -292,7 +294,61 @@ func (m *kubeGenericRuntimeManager) KillPod(pod *api.Pod, runningPod kubecontain
 // GetPodStatus retrieves the status of the pod, including the
 // information of all containers in the pod that are visble in Runtime.
 func (m *kubeGenericRuntimeManager) GetPodStatus(uid kubetypes.UID, name, namespace string) (*kubecontainer.PodStatus, error) {
-	return nil, fmt.Errorf("not implemented")
+	// Now we retain restart count of container as a docker label. Each time a container
```

Comment:
I'm not sure that this comment makes sense here. This area doesn't really have anything to do with this particular hack. I don't feel strongly, just suspect it would fit better somwhere different